src/plugins/index.js

```diff
@@ -50,24 +62,62 @@ export function TextContent(options={}) {
  *
  * This also includes {@link Content}
  *
- * @param {object} options.content options for the {@link Content} plugin
- * @param {object} options.list    options for `slate-edit-list`
+ * @param {string[]} options.content additional block node types allowed in
+ *                                   text-block content
+ * @param {string[]} options.document_content additional block node types
+ *                                            allowed as children of section
+ *                                            and document
+ * @param {string[]|null} options.marks List of additional mark types allowed
+ *                                      in text content.
+ * @param {object} options.list options for `slate-edit-list`
  */
 export function Document(options={}) {
     const {
-        content = {},
         list = {},
     } = options
 
+    const content = [
+        ...(options.content || []),
+        'paragraph',
+        'quotation',
```

Comment:
I'm not sure why `quotation` and `code` are here. `code` maybe make sens, but `quotation` is similar to `admonition`

Reply:
Quotes were already allowed in all those places before this change. And personally I would consider it weird if adding quotes to exercises or notes was not allowed.